pil_resize_image_block.py

```diff
@@ -0,0 +1,21 @@
+from nio.block.base import Block
+from nio.properties import IntProperty, VersionProperty
+
+
+class PILResizeImage(Block):
+
+    """ Use PIL to open a file and store the Image in a signal """
+
+    version = VersionProperty('0.1.0')
+    x = IntProperty(title='Width')
+    y = IntProperty(title='Height')
+
+    def process_signals(self, signals, input_id='default'):
+        for signal in signals:
+            img = signal.image
+            new_dims = (self.x(signal), self.y(signal))
+            self.logger.debug('image size: {}'.format(img.size))
+            resized = img.resize(new_dims)
```

Comment:
This doesn't need `Image.ANTIALIAS` anymore?

Reply:
Doesn't seem to, I didn't find anything in the docs that made me think I needed to include it.
https://pillow.readthedocs.io/en/3.1.x/reference/Image.html#PIL.Image.Image.resize

Reply:
maybe this should be a block property, but I avoided adding features that I wasn't using. Trying to be pragmatic and all that.